ztp/argocd/README.md

```diff
@@ -0,0 +1,88 @@
+# README #
+
+This directory provides RAN site ArgoCD gitops pipeline container image definitions
```

Comment:
you call it argocd,
but in reality there's nothing argocd specific about it IIUC. it's a container with the policygenerator
right?

Reply:
The concept of sync, pre-sync and post-sync resource hooks and examples provided are currently argocd specific